docker/bin/upload-staticfiles.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -exo pipefail
+
+GIT_COMMIT=${GIT_COMMIT:-latest}
+GIT_COMMIT_SHORT=${GIT_COMMIT_SHORT:-$GIT_COMMIT}
+CONTAINER_NAME="kitsune-static-${GIT_COMMIT}"
+IMAGE_NAME="mozilla/kitsune:prod-${GIT_COMMIT_SHORT}"
+TMP_DIR="s3-static"
+TMP_DIR_HASHED="s3-static-hashed"
+
+rm -rf "${TMP_DIR}"
+rm -rf "${TMP_DIR_HASHED}"
+
+# extract the static files
+docker create --name "${CONTAINER_NAME}" "${IMAGE_NAME}"
+docker cp "${CONTAINER_NAME}:/app/static" "${TMP_DIR}"
+docker rm -f "${CONTAINER_NAME}"
+
+# separate the hashed files into another directory
+mkdir "${TMP_DIR_HASHED}"
+find ${TMP_DIR} -maxdepth 1 -type f -regextype sed -regex ".*\.[0-9a-f]\{20\}\..*" -exec mv -t ${TMP_DIR_HASHED} {} +
```

Comment:
With the changes in https://github.com/mozilla/kitsune/commit/b3371572302189fd864fbdcdbba21ed3193b74ee, this needs to be updated to `".*\.[0-9a-f]\{16\}\..*"`, or were you going to revert all the whitenoise commits in this PR too?

Reply:
no let's keep everything as it is and I will update this commit